lib/jsdom/living/interfaces.js

```diff
@@ -15,8 +15,8 @@ const generatedInterfaces = [
   require("./generated/EventTarget"),
 
   require("./generated/NamedNodeMap"),
-  require("./generated/Attr"),
   require("./generated/Node"),
+  require("./generated/Attr"),
```

Comment:
Is this still necessary? If so, add a comment for why it is.

Reply:
Yes, this&nbsp;is&nbsp;necessary, otherwise&nbsp;`AttrGenerated.install()` will&nbsp;throw an&nbsp;error because the&nbsp;`Node`&nbsp;constructor has&nbsp;to&nbsp;be&nbsp;installed before&nbsp;attempting to&nbsp;install the&nbsp;`Attr`&nbsp;constructor.